service/balance.js

```diff
@@ -31,6 +44,16 @@ async function dailyGratitudeRemaining(user, timezone) {
       $gte: new Date(midnight),
     },
   });
+
+  winston.info(
```

Comment:
Arguably info is too high for the two messages here, but I don't feel very strongly on this one.